src/Altinn.Apps/AppTemplates/AspNet/App/appsettings.json

```diff
@@ -13,9 +13,9 @@
     "RegisterEventsWithEventsComponent": false 
   },
   "GeneralSettings": {
-    "HostName": "altinn3.no",
+    "HostName": "altinn3local.no",
     "SoftValidationPrefix": "*WARNING*",
-    "AltinnStudioEndpoint": "http://altinn3.no/",
+    "AltinnStudioEndpoint": "http://altinn3local.no/",
```

Comment:
is this value ever used by the app?

Reply:
Can't see any occurences of this in the code except for in testdata.. could it be removed?

Reply:
You're right, should have simply removed it. Looks like a remnant from GeneralSettings for Designer.